pyproject.toml

```diff
@@ -15,10 +15,11 @@ authors = [
 dependencies = [
     "jax>=0.1.67",
     "jaxlib>=0.1.47",
+    "jaxopt>=0.5.5",
     # https://github.com/google/jax/discussions/9951#discussioncomment-3017784
     "numpy>=1.18.4, !=1.23.0",
     "matplotlib>=3.0.0",
-    "flax>=0.5.2",
+    "flax>=0.5.2,<=0.6.4",
```

Comment:
I'd remove this pin and pin jax/flax in the GPU CI, see [here](https://github.com/ott-jax/ott/pull/244/commits/64ab7e7ddcaa5dc2e6476237532806237ea6c454).
Problem is that the server has old GPUs that need old drivers and I can't upgrade them at the moment.

Reply:
Ok, reverted